go.mod

```diff
@@ -21,6 +21,7 @@ replace (
 	github.com/opencontainers/runc => github.com/opencontainers/runc v1.1.4
 	github.com/opencontainers/runtime-spec => github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
 	github.com/opencontainers/selinux => github.com/opencontainers/selinux v1.10.1
+	github.com/rancher/remotedialer => github.com/brandond/remotedialer v0.0.0-20230401005234-6d90dd8bce5d
```

Comment:
Yeah, I just haven't updated here yet because I was fixing the tests which use broken cobbled-together assertions that hide where the actual failure is occuring

https://github.com/rancher/remotedialer/pull/50